src/AssimpWorker/worker/Worker.cpp

```diff
@@ -105,16 +106,13 @@ namespace AssimpWorker {
 	}
 
 	std::string Worker::importColladaAndStore(const std::string& filesystemPathToColladaFile) {
-		AssimpWorker::AssimpImporter importer;
-		const aiScene* scene = importer.importSceneFromFile(filesystemPathToColladaFile, log);
-		if (!scene) {
-			return "";
-		}
-		std::cout << "AssImp import completed, converting..." << std::endl;
-		std::string pathToFolder = filesystemPathToColladaFile.substr(0, filesystemPathToColladaFile.find_last_of('/')+1);
-		AiSceneImporter sceneImporter(scene, pathToFolder, log);
+		std::cout << "importColladaAndStore, filesystemPathToColladaFile: " << filesystemPathToColladaFile << std::endl;
+		std::string pathToFolder = filesystemPathToColladaFile.substr(0, filesystemPathToColladaFile.find_last_of('/') + 1);
+		Poco::URI uri(filesystemPathToColladaFile);
+		ColladaMassagerRegistry registry;
+		AssimpWorker::ColladaImporter importer = AssimpWorker::ColladaImporter(uri, log, pathToFolder, registry);
```

Comment:
Why do I need to pass pathToFolder here? The importer could figure that out just as well. Same for ColladaMassagerRegistry. As a user of the class I don't care that it needs those internally.